app/views/catalog_admin/items/_common_form_fields.html.erb

```diff
@@ -1,4 +1,6 @@
 <div class="pb-4">
+  <div class="base-errors pb-2"><%= @item.errors.where(:base).first&.full_message %></div>
```

Comment:
pourquoi .first et pas to sentence ? `@item.errors.where(:base).full_message.to_sentence`